modules/claims_api/app/controllers/claims_api/application_controller.rb

```diff
@@ -77,5 +84,15 @@ def poa_request?
       headers_to_check = ['HTTP_X_VA_SSN', 'HTTP_X_VA_Consumer-Username', 'HTTP_X_VA_Birth_Date']
       (request.headers.to_h.keys & headers_to_check).length.positive?
     end
+
+    def verify_loa
+      loa_value = header(key = 'X-VA-LOA') ? header(key) : raise_missing_header(key)
```

Comment:
wouldn't the key be nil for the true/false paths? or is key coming from somewhere else?

Reply:
also couldn't you just call `header_loa` here? does that exception it throws not cause the 401 below?

Reply:
I refactored to use the header_loa method